composer.json

```diff
@@ -75,23 +75,23 @@
 		"ci": [
 			"@ci:static"
 		],
-		"ci:composer:normalize": "php ./tools/composer-normalize --dry-run",
+		"ci:composer:normalize": "@php ./tools/composer-normalize --dry-run",
 		"ci:dynamic": [
 			"@ci:tests"
 		],
-		"ci:json:lint": "find . ! -path '*.Build/*' ! -path '*Resources/Private/node_modules/*' -name '*.json' | xargs .Build/vendor/bin/jsonlint -q",
+		"ci:json:lint": "find . ! -path '*.Build/*' ! -path '*Resources/Private/node_modules/*' -name '*.json' | xargs @php .Build/vendor/bin/jsonlint -q",
```

Comment:
For the linter, we cannot use `@php` as this does not work in pipes. So this needs to stay `php`.